jsx/Form.js

```diff
@@ -262,7 +262,7 @@ var SearchElement = React.createClass({
         <div className="col-sm-9">
           <input
             type="text"
-            name={this.props.name + '_list'}
+            name={this.props.name + '_input'}
             value={value}
             id={this.props.id}
             list={this.props.name + '_list'}
```

Comment:
line 268 is supposed to be '_list' or changed to '_input'?

Reply:
no that's supposed to reference the id of the datalist element. i renamed the input to avoid confusion

Reply:
list={this.props.name + '_list'} this is the id? im not sure if we're talking about the same line